src/app/clients/form/formSections/DeliveryInstructionsCard.tsx

```diff
@@ -21,7 +21,7 @@ const DeliveryInstructionsCard: React.FC<ClientCardProps> = ({
         <GenericFormCard
             title="Delivery Instructions"
             required={false}
-            text="Please input not more than 320 characters."
+            text="Please input not more than 380 characters."
```

Comment:
Given that we define `MAX_CHARACTERS` above, can we format it into this string rather than hard coding.